stable/appmesh-gateway/templates/gateway.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if .Values.appmesh.gateway }}
+apiVersion: appmesh.k8s.aws/v1beta2
+kind: VirtualGateway
+metadata:
+  name: {{ template "appmesh-gateway.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+{{ include "appmesh-gateway.labels" . | indent 4 }}
+spec:
+  namespaceSelector:
+    matchLabels:
+      appmesh.k8s.aws/sidecarInjectorWebhook: enabled
+  podSelector:
+    matchLabels:
+      app.kubernetes.io/name: {{ template "appmesh-gateway.fullname" . }}
+  listeners:
+    - portMapping:
```

Comment:
Would access logs to /dev/stdout be a nice default here? Also, would be nice to have an option for TLS termination (not a blocker, assume most folks will terminate @ NLB and not re-encrypt to gateway).

Reply:
@bcelenza I've added the access logs